### ISSUE TITLE
pay.rb has a broken check for balance

### ISSUE THREAD

**Issue by yegor256** [zerocracy/zold#73]
Let's fix

**Comment by yegor256** [comment]
@rultor release, tag is `0.9.8`

**Comment by 0crat** [comment]
There is an unrecoverable failure on my side. Please, submit it [here](https://github.com/zerocracy/farm/issues):

```
PID: 4@6a151509-d0d4-40fa-a569-53d8cdf9ca67, thread: pool-561-thread-1
com.zerocracy.pm.Claims[113] java.lang.IllegalStateException: Duplicate claims are not allowed in [C91QJT4CF](https://www.0crat.com/p/C91QJT4CF), can't add this XML to 2 existing ones (Close job;{job=gh:zerocracy/zold#73, reason=GitHub issue was closed},Close job;{job=gh:zerocracy/zold#73, reason=GitHub issue was closed}):
<?xml version="1.0" encoding="UTF-8"?>
<claim id="1471820685">
   <created>2018-05-27T18:09:38.195Z</created>
   <type>Close job</type>
   <token>github;zerocracy/zold;73</token>
   <author>yegor256</author>
   <params>
      <param name="job">gh:zerocracy/zold#73</param>
      <param name="reason">GitHub issue was closed</param>
   </params>
   <until>2018-05-27T18:24:38.195Z</until>
</claim>

```

[1.0-SNAPSHOT](https://github.com/zerocracy/farm): Issue: zerocracy/zold#73, Action: `closed`
